### PR TITLE
chore: add packages rules for mongoose and lab

### PR DESCRIPTION
**Pull request by jugaltheshah**
These were in my previous "legacy" Renovate config, and likely should be added back to this one. 

Lab >= 15 removes the `done` callback, requiring a _lot_ of refactoring to fix. 
Mongoose - I'm not sure, but I had tested and added each of the packages manually, so it's likely needed. Will confirm why.

### CODE REVIEW

**Comment by jugaltheshah** [comment]
So far, in my local repos I see `RedOx`, `api-horizon`, `api-kiosk`, `api-messages`, `api-command-center`, `api-admin`, and `api-providers` as using Mongoose 4.x.x. Still not sure why, but if we intend on using this renovate config on some of those, 4.x.x may be safer.

**Comment by cilindrox** [comment]
See my comment: https://github.com/pagerinc/api-core/pull/462#issuecomment-444465862

@jugaltheshah we _should be upgrading_ to mongoose >=5 and using a proper db conn handling logic. Right now, if there's a network hiccup, apps enter a zombie state due to the db driver not throwing that error.

Lab updates are required regardless in order to support node 10+

This is all tech debt that should be reflected, and having these PRs opened automatically by renovate will make these issues more apparent IMO.

IIRC @kuryaki created a Jira ticket for upgrading these.

TL;DR: node 8's out, node 10 rollout - with security and performance improvements - is being blocked by the lack of lab v5+ support.
Mongoose >=5 support is blocking automatic restarts and reliability, affecting our SLAs directly.

All of these are an urgent priority, we should not blacklist these.

**Comment by cilindrox** [comment]
Closing. Feel free to reopen.
